### ISSUE TITLE
Accessibility - Steps Screen Not Reading Content Properly

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#47]
For some reason, TalkBack is reading the content from the home screen when it should be reading the given Steps screen content. It happens when you attempt to read the screen from top to bottom in continuous reading mode.

Other times it reads through most everything quickly and then starts back at the top of the screen.

**Comment by LeHaine** [comment]
I am not able to get this to reproduce. I navigate from Homepage -> Step 1, for example, and it is reading everything from the step page. Do you have any additional details to get this to reproduce?

**Comment by jfc3-dol** [comment]
I will give it another look later today or tomorrow morning since I need to spend some time today looking at more of the web version of the application that I'm currently finding new accessibility issues with along with the iOS version too.

**Comment by jfc3-dol** [comment]
TalkBak might not be reading the home screen but it's lopping through text/content on the "Step 4" main screen more than once on our Android phone ( Samsung Galaxy 7S running Android 7.0 ). Have not tried on our tablet ( Samsung Galaxy Tab S running Android 6.0.1 ) yet.

**Comment by jfc3-dol** [comment]
When the user requests to read from the top of the screen in "Step 4" TalkBack reads all the heading and main content as text and then loops through everything twice and adds the "Previous Step" and "Next Step" buttons and the footer information the last time.

**Comment by KyleLivingstonDOL** [comment]
ZTI, is there a way to prevent the Talkback feature from looping over again? Also, @jfc3-dol is this something specific to one device // one platform or an issue in general?

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,

Looking through old mail in the "Other" tab, I noticed this issue was there and still open.

We only have two devices so not sure if it's for all of them or only the two different ones we have.

**Comment by jfc3-dol** [comment]
TalkBack is still looping around the screen, which is not what I would expect.

**Comment by jfc3-dol** [comment]
@pbhatt17 

TalkBack is still looping around the screen, which is not what I would expect. TalkBack is even reading the hidden menu items in the first general pass through the page. Also, the Topics checkboxes are all being read as "Not read" or something like that if they are shown visually on the screen or not.

**Comment by jfc3-dol** [comment]
@pbhatt17

TalkBack is still looping around the screen. It starts reading the general items on the screen as plain text. Then reads items as headings links, et. as expected. Then it reads the Bookmark button/link and after that goes back to the top of the page and reads stuff again.
